cdap-ui/app/cdap/components/PreviewData/DataView/TableContainer.tsx

```diff
@@ -43,6 +44,7 @@ const styles = (theme): StyleRules => ({
     '& :last-of-type': {
       borderRight: 0,
     },
+    '& .record-pane': { width: '100%' },
```

Comment:
can this width `100%` be on the actual component itself (preventing the use of named class)?

Reply:
I don't think so, because we don't want the width of 100% to be applied to the table view.